projects/ffmpeg.cmake

```diff
@@ -0,0 +1,105 @@
+########################################
+# ffmpeg
+xpProOption(ffmpeg)
+set(VER 2.6.2)
+set(REPO https://github.com/ndrasmussen/FFmpeg)
+set(FFMPEG_SRC_PATH ${CMAKE_BINARY_DIR}/xpbase/Source/ffmpeg)
+set(FFMPEG_DOWNLOAD_FILE ffmpeg-${VER}.tar.bz2)
+set(PRO_FFMPEG
+  NAME ffmpeg
+  WEB "FFmpeg" https://www.ffmpeg.org/ "FFmpeg website"
+  LICENSE "LGPL" https://www.ffmpeg.org/legal.html "Lesser GPL v2.1"
+  DESC "complete, cross-platform solution to record, convert and stream audio and video"
+  REPO "repo" ${REPO} "forked FFmpeg repo on github"
+  VER ${VER}
+  GIT_ORIGIN git://github.com/ndrasmussen/FFmpeg.git
```

Comment:
We have forked a few of the repos with the distributePro organization, good point that we should do the same with FFmpeg, I'll look to do that as part of the cleanup I'm doing.